Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -243,20 +249,88 @@ In calculating the SCI value, you SHOULD use the highest granularity data availa
+| Further information on report | metadata/report-information-url | MAY | URL | More information on your calculation methodology may be provided, and this SHALL be a URL to an external document or software repository.  |
+
+
+<!--
+
+- Supporting infrastructure and systems contained within the [Software Boundary](#software-boundary)
+-
+  - `CI` MUST be reported
+  - `C` SHOULD be reported, but where it is not reported, a reason MUST be provided
+  - The `R` baseline used MUST be reported
+  - If the `R` baseline used wa not selected from the  for baselines, then a reason for that choice MUST be provided
+- Metadata for the SCI value being reported
+  - Product name MUST be reported
+  - Contact details, including a name and email, for a point of contact responsible for the report MUST be provided
+  - Organization SHOULD be provided, where the report is not by an individual contributor
+  - A GUID, following a format described in [RFC4122], MUST be provided to uniquely identify this particular product, resource, or service.
+  - A software version for the software product about which this report is made, MAY be provided.
+  - The date the calculation was made, following a format described in [RFC3339], MUST be provided.
+  - More information on your calculation methodology MAY be provided, as a freetext comment, limited to 300 characters.
+  - More information on your report MAY be provided, as a URL linking to an external document or software repository.
+
+
+-->
+
+### Data format for reporting
+The format of your report MUST be machine-readable, and SHOULD be in a commonly used format. Examples of commonly used formats include YAML and JSON. Other formats can be selected, but they MUST allow for the same identifiers to be parsed.
+
+Examples of valid machine-readable submissions can be found in Appendix A.
+
+You MAY link to a human readable representation of your report in the further information field, for example, in HTML or PDF format, to provide additional context to your report, or the calculation behind your report.
+
+This human-readable representation SHALL live outside the GSF, and MAY be referenced in the metadata for your report.
+
+### Submitting a report
+Reports can be made to the `GSF`  using a [Pull Request](https://guides.github.com/introduction/flow/).
+
+Only one machine-readable submission MUST exist per unique GUID. Each GUID MUST reference and relate to only one product name.
+
+When a new SCI report is made, to replace an existing report for a unique product or entity, the pull request MUST update or replace the same file.
+
+Each report must be placed into a unique folder, named after the GUID associated with the product or entity.
+
+[Pull Requests](https://guides.github.com/introduction/flow/) should point at this repository:
+``` NOTE - ADD GH REPO URL HERE ```
```

Comment:
I was thinking to make this more human readable to structure under organisation than GUID.

Organisation/Application/GUID.... (data)

Would be nice to be able to just quickly click to Microsoft/Teams and see the latest report otherwise we would need another lookup table.

Reply:
BTW, I'm starting to think this has enough merit to be run as a separate GSF project? The reporting of SCI numbers feels as big, if not bigger, than the SCI itself. Perhaps the SCI contains details of the format of the reporting file, but the **actual** method of reporting we strip this out of the SCI itself, and we'll kickstart another project. I think it deserves more discussion and thought and don't want to rush it for the V1 release. What do you think @Sealjay?